src/sql/workbench/browser/modal/modal.ts

```diff
@@ -612,8 +612,11 @@ export abstract class Modal extends Disposable implements IThemable {
 	protected set messagesElementVisible(visible: boolean) {
 		if (visible) {
 			if (this._useDefaultMessageBoxLocation) {
-				DOM.prepend(this._modalContent!, this._messageElement!);
-				this.setInitialFocusedElement();
+				// only do the append when the messageElement doesn't have parent element.
```

Comment:
It would be better to explain WHY we only want to do it then (the code itself already tells us that it's doing this)